packages/create-llama/create-app.ts

```diff
@@ -114,6 +117,17 @@ export async function createApp({
     console.log();
   }
 
+  if (isRequireConfig((tools as string[]) || [])) {
```

Comment:
`toolsRequireConfig` should  work with undefined
```suggestion
  if (toolsRequireConfig(tools) {
```